membershipextras.php

```diff
@@ -333,6 +337,25 @@ function membershipextras_civicrm_pageRun($page) {
       'page-header'
     );
   }
+
+  $f = '_' . __FUNCTION__ . '_' . get_class($page);
```

Comment:
Interesting, but on this extension we're trying to follow a more Object Oriented approach, and delegate hook implementations to separate classes, to try to declutter this file and have each class deal with a single responsibility. A similar approach is possible with OO too, though. I think we should refactor all those functions that implement hooks in members-only extension into individual classes, but that might be something we need to double-check on with PM's, as I know this might require quite some time.